ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -689,30 +689,37 @@ def gather_event_display_context(
         full_workflow_display_module_path = f"{module_path}.display.workflow"
         try:
             display_module = importlib.import_module(full_workflow_display_module_path)
```

Comment:
we want to make sure that this is the only line in the try/except. Instead, add the new logic within the `except` path, possibly as a private helper method to reduce redundant nesting of similar except statements